app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    words = command.split()
+    first_file, second_file = words[1], words[2]
```

Comment:
Whast if command doesn't containt spaces?

Reply:
> Whast if command doesn't containt spaces?

Improved, I also wrote some test for the function, if you have spare time I'd be very thankful if you check and review them